draft-fossati-tls-attestation.md

```diff
@@ -776,14 +750,18 @@ the ClientHello.
 
 ## Passport Model
 
+The `results_proposal` and `results_request` extensions are used to negotiate
+the attested TLS protocol, and specifically the verifier identities supported by each peer. These
```

Comment:
Unsure what you mean by “attested TLS protocol”.
Is it a typo for “remote attestation protocol”?

Reply:
```suggestion
the supported Attestation Results properties, specifically the verifier identities supported by each peer. These
```
Does this sound better?

Reply:
@ionut-arm Not really, because the intent was that the TLS extensions actually negotiate the use of this whole protocol. I'll try "the protocol defined here".